test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -5628,6 +5628,26 @@ def forward(self, x):
         self.run_test(OnesModel(), x, input_names=["x"], dynamic_axes={"x": [0, 1, 2]})
         self.run_test(OnesModel(), x, remained_onnx_input_idx=[])
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    @disableScriptTest()
+    def test_zeros_with_tensorinput(self):
+        class Zero_(torch.nn.Module):
+            def forward(self, x):
+                return x, torch.zeros(x, 1)
```

Comment:
Why is this returning x? I don't think it's needed to trigger the bug that this is fixing.